Project-1/Praoject_1_final_version

```diff
@@ -0,0 +1,384 @@
+  	
+  ```
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+  ```
+
+  T-Box:
+  {
+    B ⊑ Person
+    B ⊑ M
+    M ⊑ Person
+    A ⊑ ¬M
+    A ⊑ Person
+    }
+  
+  A-Box:
+  { 
+    John : M
+    John : parent_of.Chris
+    John : B
+    Chris : M
+    Mary : ¬M
+    Mary: A
+    Alex : parent_of.Mary
+    Alex : parent_of.John
+    }
+
+  ΔI = {John, Mary, Alex, Chris}
+
+  Named Individuals:
+  JohnI = J,
+  MaryI = M,
+  AlexI = A,
+  Chris = C
+
+  Concept Assignments:
+  PersonI = {J, M, A, C},
+  MI = {J, A, C},
+  FI = {M},
+  BI = {J},
+  AI = {M}
+
+Role Assignments:
+parentofI = {(J, C), (A, J), (A, M)}
+
+  ```
+6. Explain the difference - using natural language - between the first-order prefixes:
+  ```
+  (a) ∃x∀y and ∀x∃y
+	For statements coming after ∃x∀y, there must be an x included in them while there is no necessity of them having a y. Also, for such statements to be true, they must be true in all conditions of y (T or F), but as long as they are true for at least one case of x they are fine.
+	For statements coming after ∀x∃y, there must be a y included in them while there is no necessity of them having an x. Also, for such statements to be true, they must be true in all conditions of x (T or F), but as long as they are true for at least one case of y they are fine.
+	
+  (b) ∃x∀y∃z and ∀x∃y∀z 
+  	For statements coming after ∃x∀y∃z, there must be an x and a z included in them while there is no necessity of them having a y. Also, for such statements to be true, they must be true in all conditions of y (T or F), but as long as they are true for at least one case of x and one case of z they are fine.
+	For statements coming after ∀x∃y∀z, there must be a y included in them while there is no necessity of them having an x or a y. Also, for such statements to  be true, they must be true in all conditions of x and all conditions of z (T or F), but as long as they are true for at least one case of y they are fine.
+	
+  (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  	For statements coming after ∀x∃y∀z∃w, there must be a y and a w included in them while there is no necessity of them having an x or z. Also, for such statement to be true, they must be true in all conditions of x and all conditions of z (T or F), but as long as theyr are true for at least one case of y and one case of w they are fine.
+	For statements coming after ∃x∀y∃z∀w, there must be an x and a w included in them while there is no necessity of them having a y or w. Also, for such statement to be true, they must be true in all conditions of y and all conditions of w (T or F), but as long as theyr are true for at least one case of x and one case of z they are fine.
```

Comment:
Same comments for the rest of these explanations; I'm not sure what you're trying to articulate.